qt/python/mantidqt/utils/async_qt_adaptor.py

```diff
@@ -0,0 +1,181 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+from functools import wraps
+from typing import Any, Optional, Callable
+
+from qtpy import QtCore
+from qtpy.QtCore import QObject, pyqtSignal
```

Comment:
I forget everytime that in `qtpy` the object is called `Signal` and not `pyqtSignal` 🤦🏻 
 ```suggestion
from qtpy.QtCore import QObject, Signal
```

Reply:
Weirdly this worked for me locally as I tested before pushing :confused: